src/IO_Strings.f90

```diff
@@ -1726,4 +1726,37 @@ FUNCTION str_SDK_nDecimal(r,nDecimal) RESULT(string)
 
     ENDFUNCTION str_SDK_nDecimal
 !
+!-------------------------------------------------------------------------------
+!> @brief Returns a character array by repeating a provided sample
+!> 
+  FUNCTION charBanner(charSamp,reps) RESULT(tmpChar)
+    CHARACTER(LEN=*),INTENT(IN) :: charSamp
+    INTEGER(SIK),INTENT(IN) :: reps
+    CHARACTER(LEN=reps*LEN(TRIM(charSamp))) :: tmpChar
+    tmpChar = REPEAT(TRIM(charSamp),reps)
```

Comment:
Definitely had no idea that was a function

Reply:
In fact, what does `charBanner` really do for us that `REPEAT` didn't already accomplish?  Looks like it's basically the exact same interface and functionality...

Reply:
Yeah, turns out they are.